From 412343efb52b4b702d8b8641914dc932efaa5cb0 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sat, 17 Jan 2015 16:41:30 -0800
Subject: [PATCH 3/8] MacPorts Only: Don't embed the deployment target in the
 asm when using -fno-integrated-as

http://llvm.org/bugs/show_bug.cgi?id=21636

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 374ad0a4b557d2255183fc19883856261ee80a96)
---
 lib/CodeGen/AsmPrinter/AsmPrinter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git llvm_release_39/lib/CodeGen/AsmPrinter/AsmPrinter.cpp macports_release_39/lib/CodeGen/AsmPrinter/AsmPrinter.cpp
index 272baceeed8..ca69cd99ca1 100644
--- llvm_release_39/lib/CodeGen/AsmPrinter/AsmPrinter.cpp
+++ macports_release_39/lib/CodeGen/AsmPrinter/AsmPrinter.cpp
@@ -196,7 +196,7 @@ bool AsmPrinter::doInitialization(Module &M) {
   // anyway.
   Triple TT(getTargetTriple());
   // If there is a version specified, Major will be non-zero.
-  if (TT.isOSDarwin() && TT.getOSMajorVersion() != 0) {
+  if (TT.isOSDarwin() && TT.getOSMajorVersion() != 0 && MAI->useIntegratedAssembler()) {
     unsigned Major, Minor, Update;
     MCVersionMinType VersionType;
     if (TT.isWatchOS()) {
-- 
2.11.0

